src/Bicep.Cli/Commands/PublishProviderCommand.cs

```diff
@@ -41,8 +42,8 @@ public async Task<int> RunAsync(PublishProviderArguments args)
             await ioContext.Error.WriteLineAsync("The 'publish-provider' CLI command group is an experimental feature. Experimental features should be enabled for testing purposes only, as there are no guarantees about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may be subject to breaking.");
 
             var indexPath = PathHelper.ResolvePath(args.IndexFile);
-            var inputUri = PathHelper.FilePathToFileUrl(indexPath);
-            var providerReference = ValidateReference(args.TargetProviderReference, inputUri);
+            var indexUri = PathHelper.FilePathToFileUrl(indexPath);
+            var providerReference = ValidateReference(args.TargetProviderReference, indexUri);
```

Comment:
I wonder if it would make sense to introduce a dedicated command (such as `bicep provider-bundle ...`), to package the types.json file into a tar.gz archive and update the `bicep provider-publish` command to be able to take the archive file as an input.

Reply:
I think that's a good idea! Currently this is behind an experimental flag - would you be OK with me leaving as-is for now, and thinking about this in a follow-up PR, or before removing the flag?

Reply:
Yeah for sure!